Jenkinsfile

```diff
@@ -941,6 +946,16 @@ core_logic: {
           }
         }
       }
+    },
+    'ARMv7 QEMU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/ut-armv7-qemu') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.unpack_and_init('gpu', mx_pip)
+            sh "ci/build.py --docker-registry ${env.DOCKER_CACHE_REGISTRY} -p test.arm_qemu ./runtime_functions.py run_ut_py3_qemu"
```

Comment:
Is it possible to use our docker_run utility instead?

Reply:
if you look closely, not ATM. because of the runtime_functions.sh which is hardcoded.

Reply:
Ah, gotcha. We can follow up on that in a separate PR. Can you create an issue so we don't forget it?